### PR TITLE
"Updated landing page, updated navbar, updated font-family, added ill…

### PR DESCRIPTION
**Pull request by jl98**
…ustration and logo"

### CODE REVIEW

File: node_app/views/partials/nav.hbs [hunk 1]
```diff
@@ -1,9 +1,26 @@
 <nav>
-    <div class="nav-wrapper">
-        <a href="/" class="brand-logo">BPool</a>
-        <ul id="nav-mobile" class="right hide-on-med-and-down">
-            <li><a href="/ride/new">New Ride</a></li>
-            <li><a href="/auth/facebook">Login</a></li>
+    <div class="nav-wrapper black-text">
+        <a href="/" class="brand-logo center black-text"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 498 100">
+            <defs>
+                <style>
```
Comment (shannonphu) [inline node_app/views/partials/nav.hbs]: Why not put this in the external stylesheet?

File: node_app/public/style.css [hunk 2]
```diff
@@ -3,35 +3,64 @@
...
+
+h3 {
+  margin-top: 50px;
+  font-size: 24px;
 }
 
 nav {
-  background-color: rgba(0, 0, 0, 0.5);
+  background-color: rgba(0, 0, 0, 0);
+  color: rgba(0, 0, 0, 0.8) !important;
```
Comment (shannonphu) [inline node_app/public/style.css]: Why important?

File: node_app/public/style.css [hunk 3]
```diff
@@ -3,35 +3,64 @@
...
+  font-weight: 800;
+}
+
+.nav-wrapper {
+  padding-top: 10px;
+  padding-right: 30px;
+}
+
+.brand-logo svg {
+  /* margin-left: 20px; */
```
Comment (shannonphu) [inline node_app/public/style.css]: Remove commented out code

File: node_app/public/style.css (hunk 2 above)
Comment (jl98) [reply node_app/public/style.css]: Oh, that line should actually be removed. I was trying to adjust the color of the text, and realized that materialize handled colors through classes, and CSS can't change that.

File: node_app/views/partials/nav.hbs (hunk 1 above)
Comment (jl98) [reply node_app/views/partials/nav.hbs]: It's an inline svg (for the logo), which I believe has its own styling contained inside it, so I assumed that all its properties should stay contained in one area

Comment (shannonphu) [reply node_app/views/partials/nav.hbs]: Can you give the svg container some ID and then put the styling in the external css?

Comment (jl98) [reply node_app/views/partials/nav.hbs]: Yup, lemme make that change.

**Comment by jl98** [comment]
I moved the styling over, also removed an outdated file

**Pull request merged**